### ISSUE TITLE
Implement HTML5 Audio @ 00:00 time

### ISSUE THREAD

**Issue by camchardukian** [camchardukian/fcc-react-typescript-pomodoro-clock#15]
Implement HTML5 Audio when the timers display the time as 00:00, preferably such that the following unit tests from FreeCodeCamp pass:
```
1. When a countdown reaches zero (NOTE: timer MUST reach 00:00), a sound indicating that time is up should play. This should utilize an HTML5 <audio> tag and have a corresponding id="beep".
2. The audio element with id="beep" must be 1 second or longer.
3. The audio element with id of "beep" must stop playing and be rewound to the beginning when the element with the id of "reset" is clicked.
```

**Comment by camchardukian** [comment]
This feature was implemented in PR #16. Unfortunately, we're still having difficulties getting all the unit tests to pass.

**Comment by camchardukian** [comment]
All unit tests are passing now after having merged PR #20.
